Sources/StreamChatUI/ChatChannelList/ChatChannelListVC+SwiftUI.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+@available(iOS 13.0, *)
+/// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
+public typealias ChatChannelList = ChatChannelListVC._ChatChannelList<NoExtraData>
```

Comment:
This should ideally be: `public typealias ChatChannelList = _ChatChannelListVC<NoExtraData>.View`

Reply:
Adjusted.